app/uk/gov/hmrc/gform/views/main_template.scala.html

```diff
@@ -31,10 +33,21 @@
   scriptElem: Option[Html] = None,
   frontendAppConfig: FrontendAppConfig,
   authConfig: Option[AuthConfig],
-  webChat: Option[WebChat] = None
+  webChat: Option[WebChat] = None,
+  formCategory: Option[FormCategory] = None
```

Comment:
On the second thought, proper thing to do would be to change :

```
case class FormTemplate(
  ...
  formCategory: Option[FormCategory],
  ...
)
```

to 

```
case class FormTemplate(
  ...
  formCategory: FormCategory,
  ...
)
```

Since we are doing `formTemplate.formCategory.getOrElse(Default)` all over the place.

Reply:
Done